src/assets/styles/variables.css

```diff
@@ -72,6 +74,9 @@
   --box-shadow-small-inset: inset 0 0 0 1px rgba(0,0,0,.05);
   --box-shadow-selected: 0px 0px 5px 3px #e77500;
   --box-shadow-large: 0 10px 30px 0 rgba(0, 0, 0, 0.1);
+  --card-width-small: 200px;
```

Comment:
pulling these values out as CSS variables so we can reuse them between (and even outside of) components